src/aio/aio_theme_switch.py

```diff
@@ -61,118 +59,60 @@ def __init__(
         - ThemeSwitchAIO.ids.store(aio_id)
         """
 
-        from dash_bootstrap_templates import load_figure_template
-
-        load_figure_template(dbc_themes_lowercase)
-
-        if aio_id is None:
-            aio_id = str(uuid.uuid4())
-
-        if themes is None:
-            themes = [dbc.themes.BOOTSTRAP, dbc.themes.CYBORG]
+        # init icons and switch_props
         if icons is None:
             icons = {"left": "fa fa-moon", "right": "fa fa-sun"}
-
-        switch_props = switch_props.copy()
-        if "value" not in switch_props:
-            switch_props["value"] = True
-        if "className" not in switch_props:
-            switch_props["className"] = "d-inline-block ms-1"
+        if switch_props is None:
+            switch_props = {}
+        # set "value" and "className" if they don't exist
+        switch_props.setdefault("value", True)
+        switch_props.setdefault("className", "d-inline-block ms-1")
+
+        # add fontawesome resource for the icons and dbc.css to apply Bootstrap theme to dcc, DataTable and Dash AG Grid
+        app = get_app()
+        app.config.external_stylesheets += [
+            "https://use.fontawesome.com/releases/v5.15.4/css/all.css",
+            # "https://cdn.jsdelivr.net/gh/AnnMarieW/dash-bootstrap-templates/dbc.min.css"
```

Comment:
I'd prefer not to automatically include this dbc.css stylesheet and add the dbc class to the app because it would be a breaking change if someone wasn't already using it.